server/src/metadata/field-metadata/field-metadata.module.ts

```diff
@@ -1,11 +1,36 @@
 import { Module } from '@nestjs/common';
-import { TypeOrmModule } from '@nestjs/typeorm';
+
+import {
+  NestjsQueryGraphQLModule,
+  PagingStrategies,
+} from '@ptc-org/nestjs-query-graphql';
+import { NestjsQueryTypeOrmModule } from '@ptc-org/nestjs-query-typeorm';
+
+import { JwtAuthGuard } from 'src/guards/jwt.auth.guard';
 
 import { FieldMetadataService } from './field-metadata.service';
 import { FieldMetadata } from './field-metadata.entity';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([FieldMetadata], 'metadata')],
+  imports: [
+    NestjsQueryGraphQLModule.forFeature({
+      imports: [
+        NestjsQueryTypeOrmModule.forFeature([FieldMetadata], 'metadata'),
+      ],
+      resolvers: [
+        {
+          EntityClass: FieldMetadata,
+          DTOClass: FieldMetadata,
+          enableTotalCount: true,
+          pagingStrategy: PagingStrategies.CURSOR,
+          create: { disabled: true },
```

Comment:
As mentioned in the PR description, we want to enable things 1 by 1 for now. Due to how this package behave, I'll have to disable everything for now (except for the read which is the purpose of this PR).

Reply:
Note: a create actually exists but does much more than communicating with the field-metadata table and insert into it, it will also create a migration in the tenant schema and run it (see createCustomField implementation)

Reply:
@Weiko You can still use this package, allow the create and override the behaviour. Better to implement the create that way in this PR I think.
https://tripss.github.io/nestjs-query/docs/graphql/resolvers/#readresolver